### ISSUE TITLE
Ecto Guide: scrub_params Plug

### ISSUE THREAD

**Issue by mneumark** [phoenixframework/phoenix_guides#531]
The following line is no longer in the pre-generated controller:
```elixir
 plug :scrub_params, "user" when action in [:create, :update]
```

Therefore, line 424 should be changed, and any surrounding text that discusses this.

**Comment by ybur-yug** [comment]
@jeregrine I saw these ones pop up and would be happy to handle a few of them this weekend and go more in-depth. I could do all separate PR's if that is preferred or just modify the intent of this one and document it all.

**Comment by ybur-yug** [comment]
Handled in #534 (closed #532 to include all model guide updates in one PR)

**Comment by ybur-yug** [comment]
@Gazler or whomever would be the person good to close this, it is resolved with the merge of #534
